pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -79,86 +77,101 @@ In plain language, this code says:
+`paths.src.fonts` | `"./node_modules/uswds/dist/fonts"` | Source location of the USWDS fonts
+`paths.src.img` | `"./node_modules/uswds/dist/img"` | Source location of the USWDS images
+`paths.src.js` | `"./node_modules/uswds/dist/js"` | Source location of the USWDS compiled JavaScript files
+`paths.src.projectSass` | `"./sass"` | Source location of any existing project Sass files outside of `paths.dist.sass`. The `watch` script will watch this directory for changes.
+`paths.dist.sass` | `"./sass"` | Project destination for theme files (Sass entry point and settings)
+`paths.dist.img` | `"./assets/uswds/images"` | Project destination for images
+`paths.dist.fonts` | `"./assets/uswds/fonts"` | Project destination for fonts
+`paths.dist.js` | `"./assets/uswds/js"` | Project destination for compiled JavaScript
+`paths.dist.css` | `"./assets/uswds/css"` | Project destination for compiled CSS
+
+Note, the `src` settings are specific to the Design System; the `dist` settings are specific to your project.
+
+Once your paths are set, export uswds-compile functions (summarized in the following table) in your project's gulpfile.js to use them in your project:
+
+Function | Description
+--- | ---
+`compile` | `compileSass` + `compileIcons`
+`compileIcons` | Build the USWDS icon sprite into `paths.dist.img`
+`compileSass` | Compile Sass into `paths.dist.css`
+`default` | `watch`
+`copyAll` | `copySetup` + `copyAssets`
+`copyAssets` | Copies all static assets: `copyFonts` + `copyImages` + `copyJS`
+`copyFonts` | Copy USWDS fonts to `paths.dist.fonts`
+`copyImages` | Copy USWDS images to `paths.dist.img`
+`copyJS` | Copy USWDS compiled JavaScript to `paths.dist.js`
+`copySetup` | Copy USWDS theme files (Sass entry point and settings files) from the `uswds` package to `paths.dist.sass`
+`init` | `copyAll` + `compile`
+`updateUswds` | `copyAssets` + `compile`
+`watch` | Compiles, then recompiles when there are changes to Sass files in `paths.dist.sass` and `paths.src.projectSass`
+
+For any function you defined as an `export` in your `gulpfile.js`, you can run `npx gulp [function]`.
 
-{:.site-note}
-**Note**: Initialization will overwrite files already in `PROJECT_SASS_SRC`.
+## Step 5: Initialize your project
+Initialize your project to copy all the necessary image, font, and Javascript assets from the source code.
 
 Initialize your project and get started by running the following command:
 
 {:.site-terminal}
 ```bash
 npx gulp init
 ```
+If you receive the error `replaceAll is not a function` when trying to run `npx gulp init`, please verify you are using the version of Node specified in the [.nvmrc file](https://github.com/uswds/uswds/blob/main/.nvmrc) and run the command again.
 
 This command will add all the USWDS assets to the directories you set, add a project Sass entry point, and compile USWDS into CSS. Add this CSS file to the `<head>` of your project HTML.
 
```

Comment:
@mejiaj In step 6, is `npx gulp build-sass` still relevant? Should we change the recommended command to `npx gulp compile` since that is what is in our example?

Reply:
Fixed in 6291ebcb25bc6f1907713d640452fc4c710fdcdb.